bluesky/log.py

```diff
@@ -1,8 +1,9 @@
 # The LogFormatter is adapted light from tornado, which is licensed under
 # Apache 2.0. See other_licenses/ in the repository directory.
-
+import fnmatch
```

Comment:
I don't think we need support partial-matching in any of the filters here. All usages of `fnmatch` may be removed.